bench/snippets/crypto-2190.cjs

```diff
@@ -0,0 +1,52 @@
+// https://github.com/oven-sh/bun/issues/2190
+const Benchmark = require("benchmark");
```

Comment:
can you port this to use mitata

later when we add a website for our snippets we can turn it into JSON

Reply:
yeah i'll go ahead and do that